smart_kit/action/http.py

```diff
@@ -21,31 +25,106 @@ class HTTPRequestAction(Action):
         }
     """
 
-    HTTP_ACTION = BaseHttpRequestAction
+    POST = "POST"
+    GET = "GET"
+    DEFAULT_METHOD = POST
+
+    TIMEOUT = "TIMEOUT"
+    CONNECTION = "CONNECTION"
 
     def __init__(self, items, id=None):
-        self.http_action = self.HTTP_ACTION(items["params"], id)
-        self.store = items["store"]
-        self.behavior = items["behavior"]
         super().__init__(items, id)
+        self.method_params = items['params']
+        self.method_params.setdefault("method", self.DEFAULT_METHOD)
+        self.error = None
+        self.init_save_params(items)
+
+    def init_save_params(self, items):
+        self.store = items["store"]
+        self.behavior = items.get("behavior")
 
     def preprocess(self, user, text_processing, params):
         behavior_description = user.descriptions["behaviors"][self.behavior]
```

Comment:
По-моему, если не указать в `items` `behavior`, то эта строчка уронит апп.

Возможный фикс:
```suggestion
        behavior_description = user.descriptions["behaviors"].get(self.behavior)
```